pkg/constants/constants.go

```diff
@@ -43,7 +43,7 @@ const (
 	KairosDefaultArtifactName = "kairos"
 	IsoEFIPath                = "/boot/uefi.img"
 	EfiBootPath               = "/EFI/BOOT"
-	EfiLabel                  = "COS_GRUB"
+	EfiLabel                  = "COS_GRUB_LIVE"
```

Comment:
This is for a vfat image which seems to only accept max of 8 chars :D

I would change it to maybe EFI_LIVE or something like that, we dont really care that much about its label

Reply:
I think we can even drop it and dont care about it